packages/host/app/components/ai-assistant/code-block/patch-footer.gts

```diff
@@ -0,0 +1,33 @@
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+
+export interface CodeBlockPatchFooterSignature {
+  Args: {};
+  Element: HTMLElement;
+  Blocks: {
+    default: [];
+  };
+}
+
+const CodeBlockPatchFooterComponent: TemplateOnlyComponent<CodeBlockPatchFooterSignature> =
+  <template>
+    <style scoped>
+      .code-block-patch-footer {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        gap: var(--boxel-sp-xs);
+        background-color: var(--boxel-650);
+        color: var(--boxel-light);
+        padding: 8px 12px;
+        font-size: 14px;
```

Comment:
This font-size value seems to be redundant, given that the Alert component is overriding it. In general, it would be good to stick to boxel-font variables for consistency in size and line-height.